### ISSUE TITLE
Failed to refresh gradle tasks

### ISSUE THREAD

**Issue by fengyc** [microsoft/vscode-gradle#714]
Extension Name: vscode-gradle
Extension Version: 3.4.5
OS Version: macos 10.15.6
VSCode version: 1.49.2

**Describe the bug**
Got a error message, found no gradle task
```
[info] Java Home: /usr/lib/jvm/java-11-openjdk-amd64
[info] JVM Args: -XX:SoftRefLRUPolicyMSPerMB=0,-noverify,-XX:TieredStopAtLevel=1,--add-opens,java.base/java.util=ALL-UNNAMED,--add-opens,java.base/java.lang=ALL-UNNAMED,--add-opens,java.base/java.lang.invoke=ALL-UNNAMED,--add-opens,java.prefs/java.util.prefs=ALL-UNNAMED,-Xms1024m,-Xmx4048m,-Dfile.encoding=UTF-8,-Duser.country=CN,-Duser.language=zh,-Duser.variant
[info] Gradle User Home: /home/fengyc/.gradle
[info] Gradle Version: 6.6
[info] Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.6/userguide/command_line_interface.html#sec:command_line_warnings

CONFIGURE SUCCESSFUL in 7s
[error] Error getting build for /home/fengyc/workspace/cas-6.2.x: Received message larger than max (6926490 vs. 4194304)
[info] Found 0 tasks
```

**To Reproduce**
1. Download repo from https://github.com/apereo/cas/tree/6.2.x
2. Open with vscode, refresh gradle tasks

**Expected behavior**
Gradle tasks should be detected.

**Screenshots**

**Output from "Gradle Tasks"**
```
...
CONFIGURE SUCCESSFUL in 7s
[error] Error getting build for /home/fengyc/workspace/cas-6.2.x: Received message larger than max (6926490 vs. 4194304)
[info] Found 0 tasks
```

**Does the bug still exist if you disable all other extensions?**
Yes

**Additional context**

**Comment by badsyntax** [comment]
Interesting, thanks for this issue report. Can you confirm that running "./gradlew tasks --all" outside of vscode reports no issues?

**Comment by fengyc** [comment]
The output of "./gradlew tasks --all" ( very long, 26597 lines, 2723794 chars):
```
...
webapp:cas-server-webapp-jetty:verifySpringConfigurationFactories - Examine spring.factories file and ensure @Configuration classes can be located
webapp:cas-server-webapp-resources:verifySpringConfigurationFactories - Examine spring.factories file and ensure @Configuration classes can be located
webapp:cas-server-webapp-tomcat:verifySpringConfigurationFactories - Examine spring.factories file and ensure @Configuration classes can be located
webapp:cas-server-webapp-undertow:verifySpringConfigurationFactories - Examine spring.factories file and ensure @Configuration classes can be located

Rules
-----
Pattern: clean<TaskName>: Cleans the output files of a task.
Pattern: build<ConfigurationName>: Assembles the artifacts of a configuration.
Pattern: upload<ConfigurationName>: Assembles and uploads the artifacts belonging to a configuration.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.6/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 8s
1 actionable task: 1 executed
```

**Comment by badsyntax** [comment]
Looks like the fix is to increase `grpc.max_receive_message_length`

It's currently set at 4MB (4194304)

https://github.com/grpc/grpc-node/blob/227a35e89955bd5e5098ac3069237f7459b4e341/packages/grpc-js/src/constants.ts#L48

That's quite a high, this project must be huge. I'll add a fix shortly.

**Comment by badsyntax** [comment]
I've released the fix with [3.4.6](https://github.com/badsyntax/vscode-gradle/releases/tag/3.4.6). Can you test it?

**Comment by badsyntax** [comment]
Gonna close this as I believe it's fixed. Let me know if it's still a problem for you.
